addons/website/models/ir_http.py

```diff
@@ -443,11 +443,11 @@ def get_frontend_session_info(self):
         session_info.update({
             'is_website_user': request.env.user.id == request.website.user_id.id,
             'geoip_country_code': request.session.get('geoip', {}).get('country_code'),
+            'website_company_id': request.website._get_cached('company_id'),
```

Comment:
I'm not sure about the groups for `website`, but this might be too much info. I suppose that `has_group` down below has a reason

Reply:
New way to handle extra data for tours, check BOI advice below